app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -261,25 +261,48 @@
       = miq_tab_content('hawkular', 'default') do
         .form-group
           .col-md-12.col-md-12
-            = render :partial => "layouts/angular-bootstrap/endpoints_angular",
-                                 :locals  => {:ng_show                => true,
-                                              :ng_model               => "#{ng_model}",
```

Comment:
did you drop ng_model intentionally? (here and [below](https://github.com/ManageIQ/manageiq-ui-classic/pull/1205/files#diff-19776d82dcdf8bc1080fe79eb95a254bL273))